rental.rb

```diff
@@ -0,0 +1,20 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Let's add `:book and :person` to `attr_accessor` as users can write and read the them.

Reply:
This issue is well resolved! ✔️